README.md

```diff
@@ -19,6 +19,19 @@ This repo uses the Kaggle API to fetch the dataset. Follow these steps to access
 
 We also use `pre-commit` hooks to keep our code clean. Please enable this using `pre-commit install` prior to making any code changes.
 
+### Docker
+To deploy this project, a docker container was used to build, share, and run our application. 
+
+1. Install Docker
+2. Run the dockers container using `docker build . -t skin_cancer`. This will build everything in the repository.
+3. Once the docker container is build run the following command to generate a docker image and the command in the DockerFile will also launch the app: `docker run -i -d -p 8080:5000 skin_cancer`
+
+### FastAPI
+> FastAPI is a modern, fast (high-performance), web framework for building APIs with Python 3.6+ based on standard Python type hints.
+> Source: https://fastapi.tiangolo.com/
+
+Therefore, this application was created using FastAPI. In order to deploy using FastAPI and Docker, a routers directory was created. Within the routers directory, a `skin_cancer_router.py` file was created. Within the `skin_cancer` directory, an `app.py` file is needed to define the app and its router.
```

Comment:
I'm not sure if this part is necessary. It's included in the FastAPI documentation.